src/vaadin-message-list.d.ts

```diff
@@ -7,10 +7,11 @@ import { ElementMixin } from '@vaadin/vaadin-element-mixin/vaadin-element-mixin.
  *
  * ```html
  * <vaadin-message-list
- *   items='[{"text":"Hello list","time":"yesterday", "user": {"name":"Matt Mambo","abbr":"MM","colorIndex":"1"}},{"text":"Hello list","time":"right now", "user": {"name":"Linsey Listy","abbr":"LL","colorIndex":"2"}}]'>
+ *   items='[{"text":"Hello list","time":"yesterday","userName":"Matt Mambo","userAbbr":"MM","userColorIndex":1},
+ *           {"text":"Another message","time":"right now", "userName":"Linsey Listy","userAbbr":"LL","userColorIndex":2, "userImg":"/test/visual/avatars/avatar.jpg"}]'>
  * </vaadin-message-list>
  * ```
- *
+ * It is recommended to set it programmatically by calling messageList.items = [...];
```

Comment:
Consider adding an example similar to what [`vaadin-menu-bar`](https://github.com/vaadin/vaadin-menu-bar/blob/fd7ebd5d34ca18684b32e72143f844c11fdb8071/src/vaadin-menu-bar.js#L24-L29) API documentation contains.

Reply:
Done.